hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/TargetTable.java

```diff
@@ -140,11 +135,14 @@ public void onEvents(final List<?> events) {
         if (TargetCreatedEvent.class.isInstance(firstEvent)) {
             onTargetCreatedEvents();
         } else if (TargetInfoUpdateEvent.class.isInstance(firstEvent)) {
-            onTargetInfoUpdateEvents((List<TargetInfoUpdateEvent>) events);
+            onTargetUpdateEvents(((List<TargetInfoUpdateEvent>) events).stream()
```

Comment:
This pull-request is about memory consumption and lists, but here you are copying the event list into a new list, I guess I know why. Probably because of using the same method `onTargetUpdateEvents` and the same logic.  I guess here it's OK to map the lists to another list of `List<Target>` but it would also work if you have a method 3 methods like :
```
void onEvent(List<TargetInfoUpdateEvent> events)
{
   events.foreach(event->onEvent(event.getEntity())):
}

void onEvent(List<TargetUpdateEvent> events)
{
   events.foreach(event->onEvent(event.getEntity())):
}

void onEvent(Target t)
{
  // do update table
}
```